blazor/gantt-chart/taskbar-editing.md

```diff
@@ -52,4 +52,54 @@ Modify the task details through user interaction, such as resizing and dragging
         return Tasks;
     }
 }
-```
\ No newline at end of file
+```
+
+## Creating taskbar on draw action
+
+A taskbar can be drawn in an empty row by drag and drop interaction. This can be achieved by enabling [GanttEditSettings.AllowSchedulingOnDrag](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.GanttEditSettings.html#Syncfusion_Blazor_Gantt_GanttEditSettings_AllowSchedulingOnDrag) and [GanttEditSettings.AllowTaskbarEditing](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.GanttEditSettings.html#Syncfusion_Blazor_Gantt_GanttEditSettings_AllowTaskbarEditing) property.
+A taskbar can be drawn only when [AllowUnscheduledTasks](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_AllowUnscheduledTasks) property is set to true.
```

Comment:
and when a task as has no start and end date

Reply:
Modified as suggested.